### PR TITLE
Updated some Opcodes.  First time i work on that so please tell me if th...

### PR DESCRIPTION
**Pull request by danlapps**
...is this any good ? new to this

### CODE REVIEW

**Comment by DDuarte** [comment]
Source of the opcodes?

**Comment by danlapps** [comment]
some gathered from skyfire oppcode files and confirmed with sniff, other straight from sniff

sniffs retail as of yesterday

**Comment by danlapps** [comment]
Example ClientToServer: CMSG_AUCTION_LIST_OWNER_ITEMS (0x03E2) Length: 16 ConnIdx: 2 EP:  Time: 01/12/2015 14:57:16.386 Number: 2825

File: src/server/game/Server/Protocol/Opcodes.h [hunk 1]
```diff
@@ -97,14 +97,14 @@ enum OpcodeClient : uint32
     CMSG_BATTLEFIELD_MGR_ENTRY_INVITE_RESPONSE        = 0x059A,
     CMSG_BATTLEFIELD_MGR_EXIT_REQUEST                 = 0x1381,
     CMSG_BATTLEFIELD_MGR_QUEUE_INVITE_RESPONSE        = 0x122A,
-    CMSG_BATTLEFIELD_MGR_QUEUE_REQUEST                = 0xBADD,
+    CMSG_BATTLEFIELD_MGR_QUEUE_REQUEST                = 0x1381,
```
Comment (DDuarte) [inline src/server/game/Server/Protocol/Opcodes.h]: Duplicated with CMSG_BATTLEFIELD_MGR_EXIT_REQUEST

File: src/server/game/Server/Protocol/Opcodes.h [hunk 2]
```diff
@@ -72,24 +72,24 @@ enum OpcodeClient : uint32
...
-    CMSG_AUCTION_PLACE_BID                            = 0xBADD,
-    CMSG_AUCTION_REMOVE_ITEM                          = 0xBADD,
-    CMSG_AUCTION_SELL_ITEM                            = 0xBADD,
+    CMSG_AUCTION_LIST_BIDDER_ITEMS                    = 0x050E,
+    CMSG_AUCTION_LIST_ITEMS                           = 0x0B53,
+    CMSG_AUCTION_LIST_OWNER_ITEMS                     = 0x03E2,
+    CMSG_AUCTION_LIST_PENDING_SALES                   = 0x11E2,
+    CMSG_AUCTION_PLACE_BID                            = 0x0B11,
+    CMSG_AUCTION_REMOVE_ITEM                          = 0x1BEB,
+    CMSG_AUCTION_SELL_ITEM                            = 0x0027,
```
Comment (DDuarte) [inline src/server/game/Server/Protocol/Opcodes.h]: Conflicts with SMSG_EQUIPMENT_SET_SAVED 0x0027

**Comment by DDuarte** [comment]
Thanks for the PR.
I added the opcodes you pull requests to TrinityCore/WowPacketParser#158 and they will be verified soon enough.

I won't update TC with their values yet. For that to happen, the opcodes structure also has to be updated.

File: src/server/game/Server/Protocol/Opcodes.h (hunk 2 above)
Comment (Shauren) [inline src/server/game/Server/Protocol/Opcodes.h]: And it's not a problem, SMSG opcodes can have the same values as CMSG with completely different function

**Pull request closed without merging**